e2e/panels-suite/frontend-sandbox-panel.spec.ts

```diff
@@ -64,6 +77,7 @@ describe('Panel sandbox', () => {
     it('Does not add iframes to body', () => {
       // this button adds 3 iframes to the body
       cy.get('[data-testid="button-create-iframes"]').click();
+      cy.wait(100); // small delay to prevent false positives from too fast tests
```

Comment:
`cy.get` will retry for 4000 ms, so don't think you'll need this line.

Reply:
This is different, I think. Cypress doesn't retry on non-visible checks. It only retries on visible checks.

The check next to this line checks for a non-visible item. If the check between the click and the non-visible check is too fast, it may happen that the item does render but later than cypress check and we get a false positive. Same in the following place I put it.

Reply:
Gotcha :+1: